### ISSUE TITLE
Merge 'Discover' and 'Following'

### ISSUE THREAD

**Issue by ExtRIELICi** [bluesky-social/social-app#6780]
### Describe the Feature

It would be nice if we could have both in one feed, similarly to Twitter.

**Comment by surfdude29** [comment]
But... they're two very different feeds, serving different use cases? 🤔

**Comment by ExtRIELICi** [comment]
Yes, I agree. What I meant is that there should be an option in the settings to merge them and view all of them in a single feed.

**Comment by auroursa** [comment]
Discover algorithm might not be perfect at the moment, but it does currently show posts from people you follow, similar to Twitter’s default behavior.
